lib/Component/VCard.php

```diff
@@ -357,7 +357,7 @@ function getValidationRules() {
 
             // FN is commented out, because it's already handled by the
             // validate function, which may also try to repair it.
-            // 'FN'           => '+',
+            // 'FN'           => '1',
```

Comment:
Haha, isn't it kinda late (or early)? 😴

The RFC [says](https://tools.ietf.org/html/rfc6350#section-6.2.1) *1 ("One **or more** instances per vCard MUST be present"), so according to the RFC "+" is correct, but it didn't make sense to me. Why would there ever be more than one FN?

Aren't we also enforcing just 1 FN here?
https://github.com/fruux/sabre-vobject/blob/master/lib/Component/VCard.php#L269

Happy to revert this one though =) Let me know.